packages/lesmis/src/views/DashboardView.js

```diff
@@ -57,123 +57,71 @@ const getProfileLabel = entityType => {
+      TabComponent: TabTemplate,
+      componentProps: {
+        Body: 'Free Indicator Selection',
+      },
     },
-  },
-  {
-    value: 'essdpPrimary',
-    label: 'ESSDP Primary sub-sector',
-    TabComponent: DashboardReportTabView,
-    componentProps: {
-      filterSubDashboards: ({ dashboardCode }) =>
-        dashboardCode.startsWith(DASHBOARD_CODES.essdpPrimary),
+    {
+      value: 'ESSDP_Plan',
+      label: 'ESSDP Plan 2021-25 M&E Framework',
+      TabComponent: TabTemplate,
+      componentProps: {
+        Body: '9th Education Sector and Sports Development Plan 2021-25 M&E Framework',
+        filterSubDashboards: ({ dashboardCode }) => dashboardCode.startsWith('LESMIS_ESSDP_Plan'),
+      },
     },
-  },
-  {
-    value: 'essdpLowerSecondary',
-    label: 'ESSDP Lower secondary sub-sector',
-    TabComponent: DashboardReportTabView,
-    componentProps: {
-      filterSubDashboards: ({ dashboardCode }) =>
-        dashboardCode.startsWith(DASHBOARD_CODES.essdpLowerSecondary),
-    },
-  },
-  {
-    value: 'essdpUpperSecondary',
-    label: 'ESSDP Upper secondary sub-sector',
-    TabComponent: DashboardReportTabView,
-    componentProps: {
-      filterSubDashboards: ({ dashboardCode }) =>
-        dashboardCode.startsWith(DASHBOARD_CODES.essdpUpperSecondary),
-    },
-  },
-  {
-    value: 'emergency',
-    label: 'Emergency in Education Preparedness and Response',
-    TabComponent: DashboardReportTabView,
-    componentProps: {
-      filterSubDashboards: ({ dashboardCode }) =>
-        dashboardCode.startsWith(DASHBOARD_CODES.emergencyInEducation),
-    },
-  },
-  {
-    value: 'internationalSDGs',
-    label: 'International reporting on SDGs',
-    TabComponent: DashboardReportTabView,
-    componentProps: {
-      filterSubDashboards: ({ dashboardCode }) =>
-        dashboardCode.startsWith(DASHBOARD_CODES.internationalSDGs),
-    },
-  },
-];
+    ...SUB_DASHBOARD_OPTIONS.map(dashboard => ({
+      value: dashboard.code,
+      label: dashboard.label,
+      TabComponent: DashboardReportTabView,
+      componentProps: {
+        filterSubDashboards: ({ dashboardCode }) =>
+          dashboardCode.startsWith(`LESMIS_${dashboard.code}`),
+      },
+    })),
+  ];
 
-// Gets the best default dashboard possible, and check if the selected dashboard is valid
-const useDefaultDashboardTab = (selectedDashboard = null, options) => {
-  if (!options || options.length === 0) {
-    return null;
-  }
+  // default option
+  let selectedOption = dropdownOptions[0];
 
-  if (selectedDashboard && options.find(option => option.value === selectedDashboard)) {
-    return selectedDashboard;
+  // check if the selected dashboard is valid
+  if (selectedDashboard && dropdownOptions.find(option => option.value === selectedDashboard)) {
+    selectedOption = dropdownOptions.find(option => option.value === selectedDashboard);
   }
 
-  return options[0].value;
+  return { dropdownOptions, selectedOption };
```

Comment:
Just because there's nothing else to pick up on in this PR, here's a suggestion to avoid the `let` and the double `.find`
```
const validSelectedOption = selectedDashboard && dropdownOptions.find(option => option.value === selectedDashboard);
return { 
  dropDownOptions, 
  selectedOption: validSelectedOption || dropdownOptions[0],
};
```

Reply:
Nice!